bin/ccs-push

```diff
@@ -118,7 +118,7 @@ else
         mkdir -p $staging_root && \
         mkdir -p $oods_landing_root && \
         cat > $staging_root/$remote_file && \
-        ln $staging_root/$remote_file $oods_landing_root/$remote_file && \
+        mv $staging_root/$remote_file $oods_landing_root/$remote_file && \
         rm $staging_root/$remote_file" <"$filename"
```

Comment:
The `rm` is going to fail if the `mv` succeeds.  You need either `rm -f` to not care or just leave it out altogether.

Reply:
I'll remove it